### PR TITLE
Add event to access user_id in extensions

### PR DESCRIPTION
**Pull request by javyxx**
Checklist:

- [x] Correct branch: master for new features; 3.2.x, 3.1.x for fixes
- [ ] Tests pass
- [ ] Code follows coding guidelines: [master / 3.2.x](https://area51.phpbb.com/docs/master/coding-guidelines.html), [3.1.x](https://area51.phpbb.com/docs/31x/coding-guidelines.html)
- [ ] Commit follows commit message [format](https://wiki.phpbb.com/Git#Commit_Messages)

Tracker ticket (set the ticket ID to **your ticket ID**):

https://tracker.phpbb.com/browse/PHPBB3-12345

I need a user_id for my extension .

### CODE REVIEW

File: phpBB/includes/ucp/ucp_register.php [hunk 1]
```diff
@@ -372,6 +372,22 @@ function main($id, $mode)
...
+				* Add access to user_id 
+				*
+				* @event core.ucp_insert_user_row_after
+				* @var	bool	submit		Do we display the form only
+				*							or did the user press submit
+				* @var	array	cp_data		Array with custom profile fields data
+				* @var	array	user_row	Array with current ucp registration data
+				* @since 3.1.10-RC1
+				*/
+				$vars = array('submit', 'cp_data', 'user_row', 'user_id');
```
Comment (Elsensee) [inline phpBB/includes/ucp/ucp_register.php]: At this point `$submit` will always be true, thus providing `$submit` will be senseless. (Also, the user has been created now, so.. what's the point of that variable, really?)

**Comment by Elsensee** [comment]
Thank you for your contribution. :)

Travis fails, because the commit message doesn't fit our requirements.
Please take a look at [this wiki article](https://wiki.phpbb.com/Git#Commit_Messages) and fix your message.
Also, "Update ucp_register.php" doesn't provide any information about what is done in the commit. You added the event "core.ucp_insert_user_row_after".

And as a last note, the documentation for the event, which is required, isn't really complete. The variable `$user_id` is missing. And you *can* alter the `$user_row` but it just won't do anythinng since the user has now been created with the `$user_row` provided before.

Again, thank you for your contribution, we value that very much, but these points will help future contributors and other extension developers to maintain the codebase or write better extensions.

**Comment by javyxx** [comment]
You are welcome . I need this point for my own extensión .

**Comment by Elsensee** [comment]
Actually, why don't you use the event `core.user_add_after`?
See here: https://github.com/phpbb/phpbb/blob/f68820762b2d78bc26c59724a3dd98091bb3379b/phpBB/includes/functions_user.php#L386-L396

**Comment by javyxx** [comment]
Yes sorry. I don't see that point . Ok, I used that event .

**Comment by Elsensee** [comment]
Good, I close this PR then.

**Pull request closed without merging**